deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/layers/recurrent/LayerNormalization.java

```diff
@@ -0,0 +1,10 @@
+package org.deeplearning4j.nn.layers.recurrent;
+
+import org.nd4j.linalg.api.ndarray.INDArray;
+
+/**
+ * Created by fac2003 on 4/6/17.
+ */
+public interface LayerNormalization {
+    void normalize(INDArray activations);
```

Comment:
And I thought I was following the project unwritten guidelines for documentation :-)
Fine, I will add a doc for the interface. The @see is unnecessary when you use a modern IDE (i.e.,  click on the arrow down on an interface to get a list of implementations in IDEA).